### ISSUE TITLE
progress-spinner: update to evo

### ISSUE THREAD

**Issue by agliga** [eBay/skin#1667]
There are some progress spinner changes that came through, updating the animation and timing function.

Here's the list of changes:
```
animation: spin 1.5s linear infinite;
animation-timing-function: cubic-bezier(0.7, 0.2, 0.3, 0.73);

@keyframes spin {
        from {
            transform: rotate(0deg);
        }

        to {
            transform: rotate(720deg);
        }
}
```
Also sizes of spinner should be 24px for large and 16px respectively. 
Also new assets:
[evo-spinners.zip](https://github.com/eBay/skin/files/8045103/evo-spinners.zip)

(Note, the large part should be blue and the smaller part grey, so swapped from what it is currently).

**Comment by agliga** [comment]
![spinner](https://user-images.githubusercontent.com/1755269/153518081-41a37b96-ec49-4145-9b0e-4a95d4b24e73.gif)

Here's what it looks like

**Comment by agliga** [comment]
For now, we will just change the animation since that is simple to do. Assets should be a separate PR since this will be breaking.

**Comment by agliga** [comment]
Waiting on assets from design to see if we keep the old spinners or not

**Comment by agliga** [comment]
Final design assets:
Will have spinner small and regular which are the new assets. 
Spinner large will be the regular spinner that we currently have (30x30. We will remove the 60x60 icon)
